core/domain/feedback_domain.py

```diff
@@ -362,6 +363,36 @@ def to_dict(self) -> FeedbackMessageReferenceDict:
             'message_id': self.message_id
         }
 
+    @classmethod
+    def from_dict(
+        cls,
+        reference_dict: FeedbackMessageReferenceDict
+        ) -> FeedbackMessageReference:
+        """Return a FeedbackMessageReference domain object from a dict.
+
+        Args:
+            reference_dict: dict. The dict representation of a
+                FeedbackMessageReference object.
+
+        Returns:
+            FeedbackMessageReference. The corresponding FeedbackMessageReference
+            domain object.
+        """
+        # Investigate why reference_dict is passed as
+        # string and not FeedbackMessageReferenceDict when called from Postman.
+        if isinstance(reference_dict, str):
+            reference_dict = json.loads(reference_dict)
+        return cls(
+            reference_dict['entity_type'],
+            reference_dict['entity_id'],
+            reference_dict['thread_id'],
+            reference_dict['message_id']
+        )
+
+    def validate(self) -> None:
```

Comment:
The `normalize_and_validate` function complained FeedbackMessageReference does not have a `validate` function. For now, I added a stub. Are there specifications of how input should be formatted and what values are valid for the specific domain object?

Reply:
Yes @MattiasErlingson you can write a validate function in the FeedbackMessageReference domain class.
To know the input values, you need to look at the following things:
- Datastore class for the corresponding domain class
- Tests for the class
- Or maybe the docstring

Reply:
Ok. I'll have a look and create a first draft.

Reply:
@Nik-09, added validation for reference dict.